primitives/src/constants/ztg.rs

```diff
@@ -46,10 +46,19 @@ pub const TEAM_AND_ADVISORS: u128 = 15_000_000;
 /// Total ZTG amount for Zeitgesit foundation.
 pub const ZEITGEIST_FOUNDATION: u128 = 22_000_000;
 
+/// Total ZTG amount at genesis.
+pub const TOTAL_INITIAL_ZTG: u128 = COMMUNITY_INCENTIVES
+    + PARACHAIN_LEASE
+    + PUBLIC_SALE
+    + SEED_SALE
+    + STRATEGIC_SALE
+    + TEAM_AND_ADVISORS
+    + ZEITGEIST_FOUNDATION;
+
 // Inflation
 
 /// Perthousand liquidity mining inflation. 2%
-pub const LIQUIDITY_MINING_PTD: Perbill = Perbill::from_perthousand(20);
+pub const LIQUIDITY_MINING_PTD: Perbill = Perbill::from_perthousand(0);
 
 /// Perthousand collator staking inflation. 1.5%
-pub const STAKING_PTD: Perbill = Perbill::from_perthousand(15);
+pub const STAKING_PTD: Perbill = Perbill::from_perthousand(50);
```

Comment:
Comment above seems to need an update.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/960/commits/f499f803652a4152aa3578e3316b53c1a54fa610